src/patternfly/components/VerticalNav/docs/code.md

```diff
@@ -0,0 +1,24 @@
+## Overview
+
+The vertical navigation list contains list items, links, icons and text. Icons, along with text, must be included for the list items to align properly. Do not omit these elements. The chevron is required for list items containing subnavigation. 
```

Comment:
@mcarrano can you confirm that icons are mandatory?

Reply:
Actually I don't think the icons should be mandatory, @mattnolting @andresgalante